build/Program.cs

```diff
@@ -62,7 +49,7 @@ static void Main(string[] args)
                 foreach(var versionInfoFile in Directory.GetFiles(Directory.GetCurrentDirectory(), "*", SearchOption.AllDirectories)
                     .Where(path => path.Contains("VersionInfo.cs")))
                 {
-                    StartProcessAndReturnOutput("git", $"checkout {versionInfoFile}");
+                    Read("git", $"checkout {versionInfoFile}");
```

Comment:
Note that `Run()` could be used instead of `Read()` here, but the difference would be that the output of the git command would be seen in the console. I don't know if you deliberately want to hide it or not.